client/my-sites/checkout/checkout/index.jsx

```diff
@@ -394,7 +394,7 @@ export class Checkout extends React.Component {
 		// - does not have a receipt number but has an item in cart(as in the case of paying with a redirect payment type)
 		if ( selectedSiteSlug && ( ! isReceiptEmpty || ! isCartEmpty ) ) {
 			if ( isJetpackNotAtomic ) {
-				return `/plans/my-plan/${ selectedSiteSlug }?thank-you&install=all`;
+				return `/plans/my-plan/${ selectedSiteSlug }?thank-you`;
```

Comment:
This change we had to revert actually (https://github.com/Automattic/wp-calypso/pull/35654) so would be ace if you can put it back for now. Thanks. :-)

Reply:
Reverted in [f9dd97f](https://github.com/Automattic/wp-calypso/pull/35283/commits/f9dd97fd9d62ecabf15c95080d663d7d8ac1e073).